server/controllers/index.ts

```diff
@@ -1,5 +1,6 @@
 import DashboardController from './dashboardController'
```

Comment:
Guessing this commit is going to be rebased in later? A lot might come out if the start page goes in first anyway!